check50/__main__.py

```diff
@@ -278,87 +310,93 @@ def main():
     global SLUG
     SLUG = args.slug
 
-
+    # dev implies offline and verbose "info" if not overwritten
     if args.dev:
         args.offline = True
-        args.verbose = True
+        if not args.verbose:
+            args.verbose = "info"
 
+    # offline implies local
     if args.offline:
+        args.no_install_dependencies = True
+        args.no_download_checks = True
         args.local = True
 
-    if args.verbose:
-        # Show lib50 commands being run in verbose mode
-        logging.basicConfig(level=os.environ.get("CHECK50_LOGLEVEL", "INFO"))
-        lib50.ProgressBar.DISABLED = True
-        args.log = True
+    # Setup logging for lib50 depending on verbosity level
+    setup_logging(args.verbose)
+
+    # Warning in case of running remotely with no_download_checks or no_install_dependencies set
+    if (args.no_download_checks or args.no_install_dependencies) and not args.local:
```

Comment:
No need to check `args.no_*` here and we should instead check if `useless_args` is truthy before printing?